code/modules/power/fusion/fusion_reactions.dm

```diff
@@ -176,3 +176,35 @@ var/global/list/fusion_reactions
 	radiation = 30
 	instability = 5
 	products = list("uranium" = 10, "lead" = 10, "borosilicate glass" = 10) // Psuedoscience but here we are
+
+// More reactions to mess around with, added by ASCC, everything past this is her word salad
+/singleton/fusion_reaction/plastic_CO2
```

Comment:
I'm pretty sure plasteel is a hematite/carbon/platinum alloy at least according to mining. I might be wrong though.